onnxruntime/core/providers/coreml/builders/model_builder.h

```diff
@@ -9,7 +9,7 @@
 #include "core/providers/coreml/model/model.h"
 #include "core/providers/coreml/coreml_options.h"
 
-#if defined(COREML_ENABLE_MLPROGRAM)
+#if defined(USE_COREML)
```

Comment:
If USE_COREML is not defined why is this file in the build?

Reply:
Good point. I will remove it.